python/smqtk/utils/factors.py

```diff
@@ -6,62 +6,76 @@
 __all__ = ["sieve_of_eratosthenes", "prime_factors", "factors", "factor_pairs"]
 
 
+# Set of known prime values
+_soe_prime_cache = [2, 3]
+# Map of not-prime values and the increment after that value where the next
+# not-prime value may be found. This should only contain keys greater-than
+# the max(_prime_cache) value for finding further prime values.
+_soe_not_prime_map = {4: 2, 9: 3}
```

Comment:
See below -- start this with `{25: 5}`

Reply:
If I start with 5 as the first value to check with instead of 4, I actually need to fill into this map the knowledge of all non-times up to 25, so I'll try that.

Reply:
I lied. Needed to modify part of the loop.